Editors/vscode/src/extension.ts

```diff
@@ -1,40 +1,52 @@
-'use strict';
-import * as vscode from 'vscode';
-import * as langclient from 'vscode-languageclient';
+"use strict";
```

Comment:
I have no particular opinion on the formatting changes here (single quote to double quote, changes in line wrapping, etc.), but they make it hard to see the actual changes.  Please remove them from this PR - if you think they're worthwhile, we could consider them separately.

Reply:
sure np, it heppend due to code formatting tool's default config.